.eslintrc.json

```diff
@@ -1,11 +1,12 @@
 {
   "root": true,
   "ignorePatterns": ["**/*"],
-  "plugins": ["@nrwl/nx"],
+  "plugins": ["@nrwl/nx", "lodash"],
   "overrides": [
     {
       "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
       "rules": {
+        "lodash/import-scope": [2, "method"],
```

Comment:
should this be configured to error instead of warn?

Reply:
Weird, it was showing as error.

Reply:
thats fine then, i was just basing this on their docs

Reply:
Oh it's the 2, that means error.

Reply:
I updated it to "error" so its more clear